src/ceph-disk/ceph_disk/main.py

```diff
@@ -2362,17 +2362,18 @@ def create_key(self):
         cluster = self.args.cluster
         bootstrap = self.args.prepare_key_template.format(cluster=cluster,
                                                           statedir=STATEDIR)
-        command_check_call(
+        command_with_stdin(
```

Comment:
@badone sorry, i was wrong. no. i think the intention is to make sure that the master branch does not miss any fixes in release branches. in this case, as you put, it's not a backport. so we don't need a `Conflict` section.